src/Nokia_LCD_Fonts.h

```diff
@@ -6,210 +6,238 @@
+        {0x07, 0x08, 0x70, 0x08, 0x07}  // 0x59 Y
+        ,
+        {0x61, 0x51, 0x49, 0x45, 0x43}  // 0x5a Z
+        ,
+        {0x00, 0x7f, 0x41, 0x41, 0x00}  // 0x5b [
+        ,
+        {0x02, 0x04, 0x08, 0x10,
+        0x20}  // 0x5c \ (keep this to escape the backslash)
+        ,
+        {0x00, 0x41, 0x41, 0x7f, 0x00}  // 0x5d ]
+        ,
+        {0x04, 0x02, 0x01, 0x02, 0x04}  // 0x5e ^
+        ,
+        {0x40, 0x40, 0x40, 0x40, 0x40}  // 0x5f _
+        ,
+        {0x00, 0x01, 0x02, 0x04, 0x00}  // 0x60 `
+        ,
+        {0x20, 0x54, 0x54, 0x54, 0x78}  // 0x61 a
+        ,
+        {0x7f, 0x48, 0x44, 0x44, 0x38}  // 0x62 b
+        ,
+        {0x38, 0x44, 0x44, 0x44, 0x20}  // 0x63 c
+        ,
+        {0x38, 0x44, 0x44, 0x48, 0x7f}  // 0x64 d
+        ,
+        {0x38, 0x54, 0x54, 0x54, 0x18}  // 0x65 e
+        ,
+        {0x08, 0x7e, 0x09, 0x01, 0x02}  // 0x66 f
+        ,
+        {0x0c, 0x52, 0x52, 0x52, 0x3e}  // 0x67 g
+        ,
+        {0x7f, 0x08, 0x04, 0x04, 0x78}  // 0x68 h
+        ,
+        {0x00, 0x44, 0x7d, 0x40, 0x00}  // 0x69 i
+        ,
+        {0x20, 0x40, 0x44, 0x3d, 0x00}  // 0x6a j
+        ,
+        {0x7f, 0x10, 0x28, 0x44, 0x00}  // 0x6b k
+        ,
+        {0x00, 0x41, 0x7f, 0x40, 0x00}  // 0x6c l
+        ,
+        {0x7c, 0x04, 0x18, 0x04, 0x78}  // 0x6d m
+        ,
+        {0x7c, 0x08, 0x04, 0x04, 0x78}  // 0x6e n
+        ,
+        {0x38, 0x44, 0x44, 0x44, 0x38}  // 0x6f o
+        ,
+        {0x7c, 0x14, 0x14, 0x14, 0x08}  // 0x70 p
+        ,
+        {0x08, 0x14, 0x14, 0x18, 0x7c}  // 0x71 q
+        ,
+        {0x7c, 0x08, 0x04, 0x04, 0x08}  // 0x72 r
+        ,
+        {0x48, 0x54, 0x54, 0x54, 0x20}  // 0x73 s
+        ,
+        {0x04, 0x3f, 0x44, 0x40, 0x20}  // 0x74 t
+        ,
+        {0x3c, 0x40, 0x40, 0x20, 0x7c}  // 0x75 u
+        ,
+        {0x1c, 0x20, 0x40, 0x20, 0x1c}  // 0x76 v
+        ,
+        {0x3c, 0x40, 0x30, 0x40, 0x3c}  // 0x77 w
+        ,
+        {0x44, 0x28, 0x10, 0x28, 0x44}  // 0x78 x
+        ,
+        {0x0c, 0x50, 0x50, 0x50, 0x3c}  // 0x79 y
+        ,
+        {0x44, 0x64, 0x54, 0x4c, 0x44}  // 0x7a z
+        ,
+        {0x00, 0x08, 0x36, 0x41, 0x00}  // 0x7b {
+        ,
+        {0x00, 0x00, 0x7f, 0x00, 0x00}  // 0x7c |
+        ,
+        {0x00, 0x41, 0x36, 0x08, 0x00}  // 0x7d }
+        ,
+        {0x10, 0x08, 0x08, 0x10, 0x08}  // 0x7e ~
+        ,
+        {0x78, 0x46, 0x41, 0x46, 0x78}  // 0x7f DEL
+    };
+
+    
+
+    struct DefaultFont: public CustomFont {
+        const unsigned char* getFont(char character) {
+            return kDefault_font[character - 0x20];
+        }
+
+        const uint8_t colSize() {
+            return kColumns_per_character;
+        }
+
+        const uint8_t rowSize() {
+            return kRows_per_character;
+        }
+        
+        const unsigned char* HSpacingChar() {
+            return hSpace;
+        }
+        
+        const uint8_t HSpacingSize() {
+            return 1;
+        }
```

Comment:
See previous suggestion that avoids inheritance and only requires a user to supply another font table.